test/plugin_functional/plugins/core_plugin_deep_links/public/plugin.tsx

```diff
@@ -29,7 +29,7 @@ export class CorePluginDeepLinksPlugin
         },
         {
           id: 'pageA',
-          title: 'DL Page A',
+          title: 'DL page A',
```

Comment:
The test plugins and test suites themselves weren't using consistent case, making it hard to debug if the tests fail. Inconsistent case can also easily be missed in future refactors so keep it simple: Use consistent case.